src/main/java/seedu/duke/Main.java

```diff
@@ -78,22 +72,49 @@ public static void main(String[] args) throws IOException, TaskManagerException
                 break;
             case "update":
                 try {
-                    updateManager(scanner, weekView, inMonthView, taskManager);
-                } catch (TaskManagerException | DateTimeParseException e) {
+                    String[] parts = input.split(",\\s*");
+                    if (parts.length < 4) {
+                        throw new TaskManagerException("Invalid input format. Please provide input in the format: " +
+                                "update, <day>, <taskIndex>, <newDescription>");
+                    }
```

Comment:
Generally well documented error handling for the user to be able to quickly correct their format